flowmachine/flowmachine/features/utilities/histogram_aggregations.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ...core.query import Query
+from typing import List
+
+
+class HistogramAggregation(Query):
+    """ 
+
+    """
+
+    def __init__(self, *, locations, bins, ranges: tuple = None):
```

Comment:
Can we change locations to metric or similar? This is for use with metrics, rather than locations.